charts/cosmic-config-server/templates/_cosmic-config-server.json.tpl

```diff
@@ -0,0 +1,18 @@
+{{- define "secrets.cosmic-config-files.cosmic-config-server.json" -}}
```

Comment:
With this new chart structure, I'd suggest to review our template naming strategy.